components/sensor/aht10.rst

```diff
@@ -4,10 +4,10 @@ AHT10 Temperature+Humidity Sensor
 .. seo::
     :description: Instructions for setting up AHT10 temperature and humidity sensors
     :image: aht10.jpg
-    :keywords: aht10 aht20 aht21 dht20
+    :keywords: aht10 aht20 dht20
 
 The ``aht10`` Temperature+Humidity sensor allows you to use your AHT10
-(`datasheet <http://www.aosong.com/userfiles/files/media/aht10%E8%A7%84%E6%A0%BC%E4%B9%A6v1_1%EF%BC%8820191015%EF%BC%89.pdf>`__), AHT20 (`datasheet <https://cdn-learn.adafruit.com/assets/assets/000/091/676/original/AHT20-datasheet-2020-4-16.pdf?1591047915>`__), AHT21 (`datasheet <https://asairsensors.com/wp-content/uploads/2021/09/Data-Sheet-AHT21-Humidity-and-Temperature-Sensor-ASAIR-V1.0.03.pdf>`__) :ref:`I²C <i2c>`-based sensor with ESPHome.
```

Comment:
Why remove the aht21 stuff?

Reply:
According to the AHT21 datasheet, the code needs to check 3 registers, and that is not part of the existing code.
![image](https://github.com/esphome/esphome-docs/assets/5025185/c62f670b-d861-4f68-b472-a6878cb6306e)

I don't think the code ever supported anything other than AHT10 originally so the documentation was wrong. The code changes I made were only to add support for AHT20.  I think its better to be omit AHT21 from the docs unless someone is able to test it.